Add first part of bottom sheet docs (#1203)
telerik/maui-docs: controls/bottomsheet/styling.md

---
title: Styling
page_title: .NET MAUI BottomSheet Documentation - Styling

> description: Learn more about how to style the handle in the Telerik UI for .NET MAUI BottomSheet.

Reviewer: From what I see, you can style more than just the handle.
```suggestion
description: Learn how to customize the appearance of the Telerik UI for .NET MAUI BottomSheet control and its handle component using styling properties like colors, borders, and dimensions.
```

position: 15
slug: bottomsheet-styling
---

# .NET MAUI BottomSheet Styling

> The Telerik UI for .NET MAUI BottomSheet control provides the following Style properties for customizing its look:

Reviewer: ```suggestion
The BottomSheet control offers styling options for both the main container and its draggable handle component. You can customize colors, borders, corner radius, and dimensions to match your application's design requirements and create a polished user experience.

The Telerik UI for .NET MAUI BottomSheet control provides the following Style properties for customizing its look:
```

* `BackgroundColor` (`Color`)&mdash;Specifies the background color of the control.
* `BorderColor` (`Color`)&mdash;Specifies the border color around the control.
* `BorderBrush` (`Brush`)&mdash;Specifies the border brush around the control.
* `BorderThickness` (`Thickness`)&mdash;Specifies the border thickness around the control.
* `CornerRadius` (`Thickness`)&mdash;Specifies the corner radius of the border around the control.

Here is an example of the BottomSheet styling.

**1.** Define the BottomSheet in XAML:

<snippet id='bottomsheet-style' />

**2.** Define the `RadBottomSheet` style in page's resources:

<snippet id='bottomsheet-style-resource' />

**3.** Add the `telerik` namespace:

```XAML
xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
```

> For a runnable example with the BottomSheet Style scenario, see the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}) and go to **BottomSheet > Styling** category.

## Handle Styling

> You can style the handle by setting the `HandleStyle` property to the `RadBottomSheet`. The target type of the `HandleStyle` is `BottomSheetHandle`.

Reviewer: ```suggestion
The handle of the BottomSheet is a small visual indicator at the top of the control that users can grab to drag and resize the panel. You can style the handle by setting the `HandleStyle` property to the `RadBottomSheet`. The target type of the `HandleStyle` is `BottomSheetHandle`.
```
